### ISSUE TITLE
hide icon?

### ISSUE THREAD

**Issue by MustafaHilmiYAVUZHAN** [Zingzy/hPyT#22]
How do we hide the icon? When you hide everything, it disappears. It stays when you hide maximize and minimize. Do you have a separate code for that?

**Comment by MustafaHilmiYAVUZHAN** [comment]
I mean only title and close button unhide

**Comment by Zingzy** [comment]
Hello @MustafaHilmiYAVUZHAN 
You see hPyT is a python wrapper of the official windows DWM API and it hence it is restricted by it. I keep scanning the API for new features but sadly couldn't find any ways to just hide the icon or the close button :/

**Comment by littlewhitecloud** [comment]
![image](https://github.com/user-attachments/assets/99f95148-f84d-43e2-91c4-b1df52b70525)
Window like this? @MustafaHilmiYAVUZHAN

**Comment by littlewhitecloud** [comment]
![image](https://github.com/user-attachments/assets/464e43b1-afed-4d78-b416-0685a48251e6)
You mean window like this? @MustafaHilmiYAVUZHAN

**Comment by MustafaHilmiYAVUZHAN** [comment]
Yes bro

**Comment by Zingzy** [comment]
> ![image](https://private-user-images.githubusercontent.com/71159641/361741909-464e43b1-afed-4d78-b416-0685a48251e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ3ODQzNTMsIm5iZiI6MTcyNDc4NDA1MywicGF0aCI6Ii83MTE1OTY0MS8zNjE3NDE5MDktNDY0ZTQzYjEtYWZlZC00ZDc4LWI0MTYtMDY4NWE0ODI1MWU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI3VDE4NDA1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcyNjNlMWFmM2E3NWQ2NmJmMWQ2ZmQ1M2RiNzRmMzc2YTliZmY4MzNlYTUwZjVkODMwNWZmMmFlMTQ3M2ZiODYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.GMK3H0GI-SFqx_t42ooaK9Ng0tRnHRcNHaetJv97m44) You mean window like this? @MustafaHilmiYAVUZHAN

This is very much possible, but this changes the whole title bar, it hides the maximize/minimize buttons and also alters the close buttons.

**Comment by Zingzy** [comment]
> ![image](https://private-user-images.githubusercontent.com/71159641/361741909-464e43b1-afed-4d78-b416-0685a48251e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ3ODQzNTMsIm5iZiI6MTcyNDc4NDA1MywicGF0aCI6Ii83MTE1OTY0MS8zNjE3NDE5MDktNDY0ZTQzYjEtYWZlZC00ZDc4LWI0MTYtMDY4NWE0ODI1MWU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI3VDE4NDA1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcyNjNlMWFmM2E3NWQ2NmJmMWQ2ZmQ1M2RiNzRmMzc2YTliZmY4MzNlYTUwZjVkODMwNWZmMmFlMTQ3M2ZiODYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.GMK3H0GI-SFqx_t42ooaK9Ng0tRnHRcNHaetJv97m44) You mean window like this? @MustafaHilmiYAVUZHAN

This is a toolWindow title bar and it lacks many features of the modern windows title bar and is originally meant to be used in docked apps or dialog boxes.

**Comment by littlewhitecloud** [comment]
![image](https://github.com/user-attachments/assets/3be2a40e-3f0e-4406-b64e-187a8df6c7e8)
The only way to hide the icon is to customize your titlebar, specific tkinter window.

**Comment by MustafaHilmiYAVUZHAN** [comment]
How do we do it? Are we going to delete the entire window and code it from scratch?

**Comment by littlewhitecloud** [comment]
> How do we do it? Are we going to delete the entire window and code it from scratch?

No, you just need to hide the titlebar and rewrite a new titlebar.
